[Feature] aviso con mensaje random para los dias feriados

IAvecilla opened the pull request:
## Motivación

La idea es hacer un mensaje personalizado para los días feriados y que no confunda a la gente que usa el discord con los avisos de clase comunes.

## Issues

Esto resuelve el issue #21

IAvecilla committed:
Chequeo de evento para clases que caen en feriado

IAvecilla committed:
Nueva embed page para clases que caen en dia feriado

IAvecilla committed:
Nuevo chequeo para detectar dias feriados
- Chequeamos que el evento no este a mas de 4 dias de distancia
- En caso de estarlo lo suponemos como que hay un feriado en el medio antes de la proxima clase

Commit:
Ran linter

Commit:
Ran linter & formatter

Commit:
Add Holiday event case

Commit:
Fix response

Commit:
Remove unused imports

josuebouchard reviewed (commented):
Genial el trabajo. Dejo un par de comentarios para debatir algunas cuestiones de diseño.

IAvecilla committed:
Uso de caracteres ascii en frases random

Co-authored-by: Josué Bouchard <7890663+josuebouchard@users.noreply.github.com>

IAvecilla committed:
Elimino linea comentada

IAvecilla committed:
Cambio en funcion para obtener frase random
- Utilizamos el largo del array de frases dentro de la funcion para evitar hardcode

IAvecilla committed:
Agregar variable extra para fechas en formato string

IAvecilla committed:
Cambio en la pagina de evento feriado
- Elimino la funcion para obtener la frase random
- La lista de frases ahora es global en el modulo
- La funcion es definida como constante y recibe la lista de frases a utilizar

Commit:
Cambio chequeo contra strings en minúscula

Commit:
Cambio chequeo contra strings en minúscula restantes

ilitteri committed:
Merge branch 'develop' into feature/mensaje_random_feriados
